packages/manager/apps/web/client/app/hosting/database/order/public/hosting-database-order-public.controller.js

```diff
@@ -0,0 +1,51 @@
+import { PRODUCT_NAME } from './hosting-database-order-public.constants';
+
+import HostingDatabaseOrderPublic from './hosting-database-order-public.service';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    OVH_MANAGER_PRODUCT_OFFERS_PRICING_CONSTANTS,
```

Comment:
Can we import those constant instead of injecting them ? That way we're slightly detach from the used framework

Reply:
@marie-j IMO, having an `import CONSTANTS from '../../../../modules/product-offers'` kills reusability because of location.
But as you mentioned it gets us linked to the framework.
So...

Reply:
Yeah, none of the solution is perfect, dependency injection independent of AngularJs would be a better way, in the meantime I'll keep it this way, for one simple reason @marie-j : every time I import the constants instead of injecting, I'm asked by others to inject them :)

Reply:
I was thinking more of `import { exportedConstant }` from '@ovh-ux/manager-product-offer'

Reply:
@marie-j yes it's what I was meaning :). I will tackle this as soon as possible

Reply:
Some constants are already exported https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/index.js#L20

Reply:
I actually did it myself ! I forgot to use it 😅

Reply:
@marie nice, I didn't know yet :)